src/Qir/Runtime/stdlib/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "qir-stdlib"
+version = "0.1.0"
```

Comment:
For now, since it is not published anywhere, the version number is not actively used. Any consumers would need to use a git depenency. I do believe that once we publish it as a crate, this should take on it's own consistent versioning scheme, but hopefully at that point it is in a separate repo such that we don't have to tie it to QDK versions.